system/system_error_monitor/CMakeLists.txt

```diff
@@ -4,11 +4,14 @@ project(system_error_monitor)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
-ament_auto_add_executable(${PROJECT_NAME}
-  src/system_error_monitor_node.cpp
+ament_auto_add_library(system_error_monitor_component SHARED
   src/system_error_monitor_core.cpp
 )
 
+rclcpp_components_register_node(system_error_monitor_component
+  PLUGIN "AutowareErrorMonitor"
+  EXECUTABLE system_error_monitor)
```

Comment:
And here

```cmake
rclcpp_components_register_node(${PROJECT_NAME}_component
  PLUGIN "AutowareErrorMonitor"
  EXECUTABLE ${PROJECT_NAME})
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/7009/commits/0e5c2df10157b0058d7022911e996757dbc526fd.